deeplake/api/dataset.py

```diff
@@ -1269,29 +1269,37 @@ def ingest_yolo(
         return ds
 
     @staticmethod
-    def ingest(
+    def ingest_classification(
         src: Union[str, pathlib.Path],
         dest: Union[str, pathlib.Path],
-        images_compression: str = "auto",
+        image_params: Optional[Dict] = None,
+        label_params: Optional[Dict] = None,
         dest_creds: Optional[Dict] = None,
         progressbar: bool = True,
         summary: bool = True,
+        num_workers: int = 0,
+        token: Optional[str] = None,
+        connect_kwargs: Optional[Dict] = None,
         **dataset_kwargs,
     ) -> Dataset:
         """Ingests a dataset from a source and stores it as a structured dataset to destination.
```

Comment:
@ProgerDav In this PR, we only handle images, correct? If so, that should be clear in the docstring